packages/git-raw-commits/test.js

```diff
@@ -227,4 +227,31 @@ describe('git-raw-commits', function () {
           }))
       })
   })
+
+  it('should allow commits to be scoped to a specific set of directories', function (done) {
+    fs.mkdirSync('./packages/common', { recursive: true })
+    writeFileSync('./packages/foo/test1', 'update')
+    exec('git add --all && git commit -m"Fifth commit"')
+    writeFileSync('test2', 'update')
+    exec('git add --all && git commit -m"Sixth commit"')
+    writeFileSync('./packages/common/test1', 'hello')
+    exec('git add --all && git commit -m"Seventh commit"')
+
+    let chunks = []
+    gitRawCommits({
+      path: ['./packages/foo', './packages/common'],
+    })
+      .pipe(through(function (chunk, enc, cb) {
+        chunks.push(chunk.toString())
```

Comment:
📝 I considered updating the above tests to use this simpler to read format, but felt that would clutter this PR.

I also temporarily updated each test to be able to run in isolation, but that added several seconds to the suite.  I strongly dislike the idea that each test requires the previous one.  Given that the total runtime is still under 10 seconds, that still might be worth doing in a  follow-up PR.